### PR TITLE
Bump version to 0.2.0

### PR DESCRIPTION
**Pull request by ManaSugi**
Signed-off-by: Manabu Sugimoto <Manabu.Sugimoto@sony.com>

### CODE REVIEW

**Comment by ManaSugi** [comment]
Oops, I forgot to update `CHANGELOG`. I'll update it right now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,16 @@ All notable changes to this project will be documented in this file.
...
+## [Unreleased]
+### Added
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+## [0.2.0] - 2021-12-31
```
Comment (rusty-snake) [inline CHANGELOG.md:17]: Nice date for a release :smile:

**Review by rusty-snake — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [Unreleased]
```
Comment (rusty-snake) [inline CHANGELOG.md:8]: The brackets in keepachangelog are just links.

Comment (ManaSugi) [reply CHANGELOG.md:8]: I referred to https://keepachangelog.com/en/1.0.0/. The example uses the brackets for Unreleased.
What do you think?

Comment (rusty-snake) [reply CHANGELOG.md:8]: They are reference links (markdown), scroll to the end of the "file".

Comment (ManaSugi) [reply CHANGELOG.md:8]: Oops, That's right. I'll change Unreleased and 0.2.0. In addition, I'll remove `Deprecated`.

**Review by rusty-snake — APPROVED** [review]

**Comment by ManaSugi** [comment]
@rusty-snake Thank you for your great contributions 😄 !

**Pull request merged**